kpi/models/asset_file.py

```diff
@@ -36,7 +36,14 @@ class AssetFile(OpenRosaManifestInterface, models.Model):
     )
 
     ALLOWED_MIME_TYPES = {
-        FORM_MEDIA: ('image', 'audio', 'video', 'text/csv', 'application/xml'),
+        FORM_MEDIA: (
+            'image',
+            'audio',
+            'video',
+            'text/csv',
+            'application/xml',
+            'application/zip',
```

Comment:
What about adding the `application/x-zip-compressed` too?

Reply:
I saw that the result of `guess_type('something.zip')` in the kpi container results in `application/zip`. It was actually the same for kobocat...